src/main/java/by/Starleken/services/interfaces/CommentService.java

```diff
@@ -0,0 +1,8 @@
+package by.Starleken.services.interfaces;
```

Comment:
Usually package hierarchy is next (interfaces are first)

```
java.by.{project}
    -> service
         MyService.java // inerface
         -> impl
              MyServiceImpl.java // implementation
```

It's not necessary to update it in this project. Just four your information